reframe/frontend/statistics.py

```diff
@@ -313,12 +313,15 @@ def performance_report(self):
             for key, ref in t.check.perfvalues.items():
                 var = key.split(':')[-1]
                 val = ref[0]
-                try:
-                    unit = ref[4]
-                except IndexError:
-                    unit = '(no unit specified)'
+                unit = ' ' + ref[4] if ref[4] else ''
+                ref_val = ref[1]
+                if ref_val:
+                    percent = round(val / ref_val * 100)
```

Comment:
I think that we should handle the case, where `ref_val` is `0`.
What do you think?

Reply:
I believe this is already handled since `bool(0) == False`; in this case the comparison will be skipped altogether.
However, maybe I should edit the condition/add a comment to make it more obvious that this case is handled?

Reply:
Yes and no. What about the case when the reference value is actually zero? On the other hand, as I mentioned in my other comment, we need a way to distinguish the "unknown reference" situation. By convention the unknown reference is the tuple `(0, None, None)`, so in that case now reference data should be printed.

Reply:
I'm not sure how common the case where the reference value is zero would be. I guess it _could_ be useful in theory to know that the reference value is zero, I'm just not sure about in practice. But if you think it's something that should be handled then I'm happy to add it!

In this case, maybe the details section could be:
```text
(ref. = 0 units)
```
…or maybe this? (more consistent with the ref ≠ 0 case)
```text
(N/A of 0 units)
```

Reply:
Also, @vkarak am I correct in understanding that the way to distinguish the unknown reference from the zero reference is by checking whether the upper/lower bounds are `None`?